src/librustdoc/html/format.rs

```diff
@@ -442,7 +442,7 @@ impl clean::GenericArgs {
     }
 }
 
-crate fn href(did: DefId, cx: &Context<'_>) -> Option<(String, ItemType, Vec<String>)> {
+crate fn href(did: RealDefId, cx: &Context<'_>) -> Option<(String, ItemType, Vec<String>)> {
```

Comment:
Hmm, it seems odd to me this would be real - can't rustdoc generate links to  impls? Or is that only intra-doc links?